CRM/Contact/Form/Search/Custom/Group.php

```diff
@@ -260,6 +260,16 @@ public function from() {
       }
       $includedGroups = implode(',', $allGroups);
 
+      /*
+       * CRM-15049 - Bug Fix for Include - Exclude Search
+       * Finding child group if exists for selected group ids
+       */
+      $childGroupIds = CRM_Contact_BAO_Group::getChildGroupIds($this->_includeGroups);
+      if (count($childGroupIds) > 0) {
+        $this->_includeGroups = array_merge($this->_includeGroups, $childGroupIds);
+      }
+      //-- End - CRM-15049 - Bug Fix for Include - Exclude Search--//
```

Comment:
Can you please remove this end comment line from the fix. Also I think a simple one line would be enough to make this understand of what this code is doing. Something like -

    //CRM-15049 - Include child group ids.
    <Fix for including child groups>